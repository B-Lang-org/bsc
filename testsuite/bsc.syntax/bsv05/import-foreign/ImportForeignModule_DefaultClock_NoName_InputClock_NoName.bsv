import "BVI" MOD =
module mkMod () ;

   default_clock (CLK1) <- exposeCurrentClock ;
   no_reset ;

   input_clock (CLK2) <- exposeCurrentClock ;

endmodule

