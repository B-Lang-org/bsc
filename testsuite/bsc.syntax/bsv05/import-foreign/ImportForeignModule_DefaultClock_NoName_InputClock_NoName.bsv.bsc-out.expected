checking package dependencies
=== parsed:
package ImportForeignModule_DefaultClock_NoName_InputClock_NoName;
module mkMod(Empty);
  let _bv1 <- exposeCurrentClock;
  let _bv0 <- exposeCurrentClock;
  Empty _m__;
  _m__ <- liftModule((unexpected) module verilog "MOD";
		     clock _clk__1(CLK1, {-inhigh-});
		     clock _clk__2(CLK2, {-inhigh-});
			 ((ClockArg _clk__1, _bv1), (ClockArg _clk__2, _bv0))
			 {
			 }
			 SchedInfo [] [] [] []
			 No combinational paths from inputs to outputs);
endmodule: mkMod

endpackage: ImportForeignModule_DefaultClock_NoName_InputClock_NoName

-----

compiling ImportForeignModule_DefaultClock_NoName_InputClock_NoName.bsv
=== parsed:
package ImportForeignModule_DefaultClock_NoName_InputClock_NoName;
module mkMod(Empty);
  let _bv1 <- exposeCurrentClock;
  let _bv0 <- exposeCurrentClock;
  Empty _m__;
  _m__ <- liftModule((unexpected) module verilog "MOD";
		     clock _clk__1(CLK1, {-inhigh-});
		     clock _clk__2(CLK2, {-inhigh-});
			 ((ClockArg _clk__1, _bv1), (ClockArg _clk__2, _bv0))
			 {
			 }
			 SchedInfo [] [] [] []
			 No combinational paths from inputs to outputs);
endmodule: mkMod

endpackage: ImportForeignModule_DefaultClock_NoName_InputClock_NoName

-----

All packages are up to date.
